src/Components/pageComponents/auth/login/index.tsx

```diff
@@ -0,0 +1,312 @@
+'use client';
+import React, { useState, useEffect, ChangeEvent, useContext } from 'react';
+import { useFormik, FormikErrors, FormikTouched, FormikValues, FormikHelpers } from 'formik';
+import { useRouter } from 'next/navigation';
+import Link from 'next/link';
+//@ts-ignore
+import { Button, Input } from 'technogetic-iron-smart-ui';
```

Comment:
remove ts-ignore

Reply:
I can not remove @ts-ignore because button and Input are used from our npm packages without using //@ts-ignore it shows and error and code is not working properli